plugins/destination/duckdb/client/client_test.go

```diff
@@ -21,10 +22,18 @@ func TestPlugin(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	p.SetLogger(zerolog.New(zerolog.NewTestWriter(t)).Level(zerolog.DebugLevel))
```

Comment:
This will help us see batchwriter errors (little bit clunky since it's not tied to each test but the whole TestPlugin context) during tests.